wdfn-server/waterdata/templates/monitoring_location_embed.html

```diff
@@ -1,5 +1,12 @@
 {% extends 'base_plain.html' %}
 
+{% block page_css %}
+ <link rel="stylesheet" href="{{ 'main.css' | asset_url }}">
```

Comment:
Please indent here and on line 7 so that it is easy to see the start and end of the block. Also put a blank line between blocks. We have been following this rule as a best practice for Jinja template blocks.

Reply:
Indented the block content.